pyQuARC/schemas/rule_mapping.json

```diff
@@ -2703,13 +2771,6 @@
                     ]
                 }
             ],
-            "dif10": [
```

Comment:
Should this stay in - looking at https://github.com/NASA-IMPACT/pyQuARC/pull/183/files#diff-5f8ce590c2cec186f5d8c7a4c4b6c825139dba048c0cb4343a71665fc25bfffdL2709